wolfcrypt/benchmark/benchmark.c

```diff
@@ -2146,53 +2300,71 @@ static void bench_stats_asym_finish_ex(const char* algo, int strength,
         opsSec = 0;
     }
 
+#ifdef BENCH_MICROSECOND
+    milliEach = each / 1000;   /* milliseconds */
+#else
     milliEach = each * 1000;   /* milliseconds */
 
+#endif
+
+#ifdef BENCH_MICROSECOND
+    digits = 5;
+#else
+    digits = 3;
+#endif
+
     SLEEP_ON_ERROR(ret);
     /* format and print to terminal */
     if (csv_format == 1) {
         /* only print out header once */
         if (asym_header_printed == 0) {
 #ifdef GENERATE_MACHINE_PARSEABLE_REPORT
     #ifdef HAVE_GET_CYCLES
-            printf("%s", "\"asym\",Algorithm,key size,operation,avg ms,ops/sec,"
-                   "ops,secs,cycles,cycles/op\n");
+            printf("%s", "\"asym\",Algorithm,key size,operation,avg ms,ops/"
+                    WOLFSSL_FIXED_TIME_UNIT "ec,ops," WOLFSSL_FIXED_TIME_UNIT
+                    "ecs,cycles,cycles/op");
     #else
-            printf("%s", "\"asym\",Algorithm,key size,operation,avg ms,ops/sec,"
-                   "ops,secs\n");
+            printf("%s", "\"asym\",Algorithm,key size,operation,avg ms,ops/"
+                    WOLFSSL_FIXED_TIME_UNIT "ec,ops," WOLFSSL_FIXED_TIME_UNIT
+                    "ecs");
     #endif
 #else
             printf("\n%sAsymmetric Ciphers:\n\n", info_prefix);
-            printf("%sAlgorithm,key size,operation,avg ms,ops/sec,\n",
-                   info_prefix);
+            printf("%sAlgorithm,key size,operation,avg ms,ops/"
+                    WOLFSSL_FIXED_TIME_UNIT "ec,", info_prefix);
 #endif
+        #ifdef ADVANCED_STATS
+            printf("max duration,min duration,mean duration,sd,\n");
```

Comment:
This is missing a comma before "max duration" when `GENERATE_MACHINE_PARSEABLE_REPORT`.  The output looks like this:
```
"asym",Algorithm,key size,operation,avg ms,ops/μsec,ops,μsecs,cycles,cycles/opmax duration,min duration,mean duration,sd,
```